### ISSUE TITLE
CPS2 game..

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kexplo/imame4all#52]
```
Greetings,

Thank for your good job! I can play the old school games in anywhere! Your MAME 
is work fine, but it's missing CPS2 game.. Could play it in your update?




```

Original issue reported on code.google.com by `smalltig...@gmail.com` on 17 Sep 2011 at 4:08

**Comment by GoogleCodeExporter** [comment]
```
Hi Sir,

Could you mind update your mame4dorid to romset 0.37b14, then I can play more 
lovely CPS2 games (Super Street fighter, Vampire Savior, Dungeons & Dragons..) 
The 0.37B14 set also is for DOS version mame and I think it may have a chance 
for running in mobile phone. Even it can using CPS2HD for play the CPS2 game 
but the UI is so bad.. Your MAME4droid UI is great! Sorry I don't know it will 
be a heavy work for you or not..But I really hope you can work for this, and 
many guys will very happy and thank for you. 


Thanks you!!!

```

Original comment by `fatcatma...@gmail.com` on 19 Sep 2011 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
Hello,
Seleuco explained many times that the MAME version cannot be changed, since 
0.37b5 is the last one with good performance on mobile platforms.
After that, MAME went to accurate vs. optimized emulation, because that's what 
they aim for; the downside is that any new games are out.
Even 0.37b6 is less optimized, and some working games will slow down/fail.

Trying to change the version anyway or adding games is a huge hassle: I tried 
adding NeoGeo games to the current list, and in the end, it didn't work since 
so many code changed between 0.37b5 (0.42) and 0.59.

However, Seleuco mentioned maybe porting the FBA2X version of Final Burn Alpha: 
http://code.google.com/p/imame4all/issues/detail?id=47
Final Burn Alpha is an emulator with a more limited set of games, but newer 
ones (NeoGeo, CPS1/2) and places speed above accuracy, so it could run those 
games easily.

TL;DR version: MAME version will not evolve (not in the next year at least), 
but something else like Final Burn Alpha could come in the meantime.
```

Original comment by `nudgenu...@gmail.com` on 19 Sep 2011 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
so what needs to be ported besides H and C drivers from new games ? MKombat was 
ported and i took a look at the source, looks mostly copied/pasted from 
original mame source, so theres no one who's interested with porting new 
drivers to this version ? With MKombat running so fast it would run all games 
fullspeed (not counting 3D ones).
```

Original comment by `2black...@gmail.com` on 13 Nov 2011 at 4:09

**Comment by GoogleCodeExporter** [comment]
```
Better off using FB Alpha not FBA2X.

Source is over here at : http://www.barryharris.me.uk/
```

Original comment by `coonstat...@gmail.com` on 7 Jan 2012 at 4:28

**Comment by GoogleCodeExporter** [comment]
```
Yes. It depends  how easy it will be... Better fba ... But maybe fba2x.... I'll 
decide when i have enought time to look the code
```

Original comment by `seleuco....@gmail.com` on 7 Jan 2012 at 7:04

**Comment by GoogleCodeExporter** [comment]
```
So this is why Super Street Fighter doesn't show in my ROM list..! :(
```

Original comment by `M.Krui...@gmail.com` on 15 Feb 2012 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
Well, screw all this, its now updated to 0.134u4. happy now?

Downside is its very complicated to get roms converted, since thats the ONLY 
way to get the roms!
```

Original comment by `thehamb...@gmail.com` on 4 Mar 2012 at 4:09

**Comment by GoogleCodeExporter** [comment]
```
Screw the conversion, just use the 0.137 romset from the old computer's mame 
section. its good for the registered users too, since its the only other set 
that isnt spam or isnt just 1 rom and isnt 37b5, and the only other set that is 
not only for "upgraded" users.
```

Original comment by `thehamb...@gmail.com` on 6 Mar 2012 at 10:10
